doc/tutorials/error_analysis/error_analysis_part1.ipynb

```diff
@@ -0,0 +1,553 @@
+    "\n",
+    "At first glance, it might seem to be very expensive to compute the SEM, because one would have to repeat the whole simulation many times. However, under the right circumstances, the SEM can be estimated from *a single series* of $N$ measurements. We will discuss how this can be done.\n",
+    "\n",
+    "### Confidence interval\n",
+    "A common confidence interval is the $95~\\%$ confidence interval. If we were given a number and a range around it, called the $95~\\%$ confidence interval, we would know that only $5~\\%$ of all samples are expected to lie outside that range. If the samples are normally distributed with a standard deviation $\\sigma$, the $95~\\%$ confidence interval would be roughly $\\pm1.96\\sigma$.\n",
+    "\n",
+    "### Interquartile range\n",
+    "The interquartile range denotes the range, within which the central $50~\\%$ of all samples lie, if one were to order them by their size. This leaves one quarter of all samples lying below the interquartile range, and another quarter of all samples above it.\n",
+    "\n",
+    "### Now &ndash; what do we use?\n",
+    "We are interested in the precision of our overall, averaged, simulation result, and not in the precision of the individual samples. Those are expected to fluctuate, and in many cases, those fluctuations are uninteresting for the end result. Out of the options presented above, the standard error of the mean is the only one doing this requirement justice. Of course, we could also multiply the SEM with $1.96$ and obtain a $95~\\%$ confidence interval for the mean. Thus, the question boils down to how to compute the SEM, which will be the topic of the rest of this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Uncorrelated samples\n",
+    "How the SEM can be computed depends on the data itself. For uncorrelated samples, it is nearly trivial:\n",
+    "\n",
+    "$\n",
+    "\\begin{align}\n",
+    "    \\bar\\epsilon = \\frac{\\bar\\sigma}{\\sqrt{N}}\\tag{2}\n",
+    "\\end{align}\n",
+    "$\n",
+    "\n",
+    "where $\\bar\\epsilon$ is the estimated SEM, $\\bar\\sigma$ is the estimated standard deviation (see eq. 1) and $N$ is the number of samples. But what does it mean for samples to be uncorrelated?\n",
+    "\n",
+    "An example for uncorrelated samples would be the rolling of a dice. The outcome of each trial is completely independent to the previous trials. We might guess any number from 1 to 6, regardless of what has been the last result. The same could be true if we ran an experiment many times independently from one another and measured a quantity each time. By looking at one experimental value, we would'nt be able to predict the next one. The best guess would be simply the mean value of the entire series.\n",
```

Comment:
This is my favourite example!! You can also directly explain correlation here. If the outcome of rolling a dice would make it more probable to get this outcome the next time you roll a dice, then the dice would produce a correlated time series.

Reply:
I worked out the correlated dice process a bit: https://github.com/jonaslandsgesell/correlated_dice/blob/master/correlated%20dice.ipynb

Reply:
Thanks for your effort. I cannot open this file. May it be private?

Reply:
I can open it in google chrome without being logged in. Let me know if the problem persists, then I can also send it to you via mail.

Reply:
Today, I was able to open it. Wow, that is a significant effort! From your first reply I imagined a short explanation of one or two extra sentences. I am not sure if *this* will fit into the existing tutorial. An intuitive picture of correlation is sufficient at this point in the tutorial. I, for one, am often struggling with texts that contain many formulas, and I imagine, some of the users would, too. However, it is a nice work, detailing many aspects of correlation, and I am thinking about a way we could include this.

Reply:
I find the example of the correlated dice very educative and just wanted to make the point that correlations can be illustrated with this exampls :) So maybe you can build up this intuition behind correlations via stating: "In the case of rolling a dice, correlations could be observed if it was (for example) more probable to obtain the same result as in the previous dice roll rather than another result."

Probably, this example is not suited to showcase the uncertainty estimation of the mean value though. Probably an AR-1 model is more suited here.